jujugui/static/gui/src/app/components/deployment-flow/_deployment-flow.scss

```diff
@@ -23,27 +23,210 @@
+            padding: 0;
+            background-color: transparent;
+        }
+
+        &-title {
+            .svg-icon {
+                margin-right: 10px;
+                vertical-align: text-top;
+            }
+
+            &-aside {
+                padding-left: 20px;
+                font-size: 16px;
+            }
+        }
+    }
+
+    &__top-action {
+        float: right;
+    }
+
+    &__label {
+        margin-bottom: 10px;
+        color: $warm-grey;
+        font-size: 0.688em;
+    }
+
+    &__row-header {
+        font-weight: 400;
+    }
+
+    &__row-header,
+    &__row {
+        border-bottom: 1px solid $light-mid-grey;
+    }
+
+    .button--neutral {
+        background-color: $white;
+    }
+
     &__clouds {
+        &-list {
+            @extend %list-style;
+        }
 
+        &-cloud {
+            background-color: $white;
+            box-shadow: $box-shadow;
+            text-align: center;
+            cursor: pointer;
+            margin-top: 20px;
+            margin-bottom: 0;
+
+            &:hover {
+                background-color: $light-grey;
+            }
+
+            &-logo {
+                display: block;
+                line-height: 120px;
+
+                .svg-icon {
+                    vertical-align: middle;
+                }
+            }
+        }
     }
 
     &__credentials {
-
+        &-form {
+            margin-top: 20px;
+        }
     }
 
     &__machines {
-
+        &-message {
+            margin: 20px 0 40px 0;
+        }
     }
 
     &__services {
+        &-budget {
+            margin-top: 40px;
+
+            &-form {
+                margin-top: 20px;
+                margin-bottom: 0;
+            }
+
+            &-increase {
+                display: inline-block;
+                margin-top: 30px;
+            }
+
+            &-chart {
+                position: relative;
+                height: 20px;
+                margin-bottom: 10px;
+
+                &-new,
+                &-limit {
+                    position: absolute;
+                    top: 0;
+                    bottom: 0;
+                    left: 0;
+                    border-radius: $border-radius;
+                }
+
+                &-limit {
+                    right: 0;
+                    background-color: $white;
+                    border: 1px solid $light-mid-grey;
+                }
+
+                &-new {
+                    width: 50%;
+                    background-color: $success;
+                }
+            }
+
+            &-indicator {
+                display: inline-block;
+                width: 15px;
+                height: 15px;
+                margin-right: 10px;
+                border-radius: $border-radius;
+                vertical-align: middle;
+
+                &--new {
+                    background-color: $success;
+                }
+
+                &--existing {
+                    background-color: #aee2b6;
```

Comment:
Is this a new colour for the colour palette? Shouldn't this use an existing colour?